client/next.js

```diff
@@ -10,13 +10,17 @@ const {
   __NEXT_DATA__: { component, props, ids, err }
 } = window
 
-const Component = evalScript(component).default
+document.addEventListener('DOMContentLoaded', () => {
```

Comment:
DOMContentLoaded is supported from IE9, I don't think we need to support IE8 anymore.